language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -157,7 +157,7 @@ public void sort(BallerinaCompletionContext context, T node, List<LSCompletionIt
             }
             if (symbol.kind() == FUNCTION || symbol.kind() == METHOD) {
                 completionItems.add(populateBallerinaFunctionCompletionItem(symbol, ctx));
-            } else if (symbol.kind() == SymbolKind.CONSTANT) {
+            } else if (symbol.kind() == SymbolKind.CONSTANT || symbol.kind() == SymbolKind.ENUM_MEMBER) {
                 CompletionItem constantCItem = ConstantCompletionItemBuilder.build((ConstantSymbol) symbol, ctx);
```

Comment:
For the enum member completion items shall we set the completion item kind as `EnumMember`

Reply:
+1

Reply:
Fixed with 00773b8